.generated_docs

```diff
@@ -23,12 +23,14 @@ docs/man/man1/kubectl-convert.1
 docs/man/man1/kubectl-create.1
 docs/man/man1/kubectl-delete.1
 docs/man/man1/kubectl-describe.1
+docs/man/man1/kubectl-drain.1
 docs/man/man1/kubectl-edit.1
 docs/man/man1/kubectl-exec.1
 docs/man/man1/kubectl-explain.1
 docs/man/man1/kubectl-expose.1
 docs/man/man1/kubectl-get.1
 docs/man/man1/kubectl-label.1
+docs/man/man1/kubectl-lame.1
```

Comment:
I really think we should reconsider the use of `lame` here -- it has connotations that some might find ableist, and I don't think it's extraordinarily descriptive anyway.  How about `rotate-node-out` and `rotate-node-in` ?